lib/commands/command.bash

```diff
@@ -8,17 +8,49 @@ if [ -z "$(declare -f -F watch_file)" ]; then
   eval "$(asdf exec direnv stdlib)"
 fi
 
+# This is inspired by https://stackoverflow.com/a/1116890
+_follow_symlink() {
+  path="$1"
+
+  # Start in the directory of the (possible) symlink.
+  cd "$(dirname "$path")"
+  filename="$(basename "$path")"
+
+  # Follow symlinks until we run out of symlinks.
+  # This probably will loop forever if there's a cycle.
+  while [ -L "$path" ]; do
+    path="$(readlink "$filename")"
+    cd "$(dirname "$path")"
+    filename="$(basename "$path")"
+  done
+
+  # Now print out the final directory we ended up in, plus the final filename.
+  echo "$(pwd -P)/$filename"
+}
+
 _load_asdf_utils() {
   if [ -z "$(declare -f -F with_plugin_env)" ]; then
-    ASDF_DIR="${ASDF_DIR:-"$(command -v asdf | xargs dirname | xargs dirname)"}"
-    # shellcheck disable=SC1090 # Can't follow non-constant source. Use a directive to specify location.
-    if [ -f "$ASDF_DIR/lib/utils.sh" ]; then
-      # shellcheck disable=SC1091
-      source "$ASDF_DIR/lib/utils.sh"
-    else
-      # shellcheck disable=SC1091
-      source "$ASDF_DIR/lib/utils.bash"
-    fi
+    ASDF_DIR="${ASDF_DIR:-"$(_follow_symlink "$(command -v asdf)" | xargs dirname | xargs dirname)"}"
+
+    utils_locations=()
+    # libexec is a Homebrew specific thing. See
+    # https://github.com/asdf-community/asdf-direnv/issues/95 for details.
+    for libdir in lib libexec/lib; do
```

Comment:
the additional look inside `libexec` looks good to me, since that's a homebrew thing, I guess there's the only way. 

Being this a bash file, do you think taking the output of `ls $ASDF_DIR/{lib,libexec}/utils.{sh,bash}` would also work ?

Reply:
I went ahead and cleaned this up using `asdf info`. I'm quite pleased with this now =)